its/ruling/src/test/resources/autoscan/autoscan-diff-by-rules.json

```diff
@@ -1191,7 +1191,7 @@
     "ruleKey": "S2259",
     "truePositives": 41,
     "falseNegatives": 0,
-    "falsePositives": 129
+    "falsePositives": 130
```

Comment:
The new FP comes from the new file that we introduced. In degraded mode, method behaviors can not be recognized (basically, almost all methods invocation are unknown methods), and so the fix can not be taken into account.

This should be handled by the following ticket: [SONARJAVA-3843](https://sonarsource.atlassian.net/browse/SONARJAVA-3843)